crates/wash-cli/tests/wash_start.rs

```diff
@@ -39,12 +41,35 @@ async fn integration_start_actor_serial() -> Result<()> {
         serde_json::from_slice(&output.stdout).context("failed to parse output")?;
     assert!(cmd_output.success, "command returned success");
 
+    // Test stopping using only aliases, yes I know this mixes stop and start, but saves on copied
+    // code
+    let status = Command::new(env!("CARGO_BIN_EXE_wash"))
+        .args([
+            "stop",
+            "actor",
+            "echo",
+            "--output",
+            "json",
+            "--timeout-ms",
+            "40000",
+            "--ctl-port",
+            &wash_instance.nats_port.to_string(),
+        ])
+        .kill_on_drop(true)
+        .stdout(Stdio::null())
+        .stderr(Stdio::null())
+        .status()
+        .await
+        .context("failed to start actor")?;
```

Comment:
Would you mind hanging this off of `impl TestWashInstance`? I think it'd be very generally useful

Reply:
There are times we are testing the output so I probably don't want to restrict it. Though it would probably be nice to have something that can generate the `Command` for you given your list of args

Reply:
Ah so I mean something [like this](https://github.com/wasmCloud/wasmCloud/blob/main/crates/wash-cli/tests/common/mod.rs#L272) so we can keep the tests small, and re-use common things to run on the actual instance.

Reply:
I get what you're saying now. I created #1032 to track doing this later as this currently matches what is done for starting the actor